lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/service/IdentityVerificationService.java

```diff
@@ -119,13 +120,17 @@ public DocumentCheckVerificationResult verifyIdentity(
                     result.setValidityScore(documentValidityScore);
                     result.setActivityHistoryScore(activityHistoryScore);
 
+                    // Data capture for VC
+                    documentCheckResult.setCheckDetails(
```

Comment:
Do we have a failed check details scenario?

Reply:
There was no failed check details, this code was only moved from [DcsThirdPartyDocumentGateway](https://github.com/alphagov/di-ipv-cri-dl-api/blob/main/lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/service/dcs/DcsThirdPartyDocumentGateway.java) so it could be common to all the api's.

Reply:
We now know this is managed in the issuecredential lambda